app/services/idv/actions/verify_document_status_action.rb

```diff
@@ -17,12 +17,15 @@ def process_async_state(current_async_state)
 
         if current_async_state.status == :done
           process_result(current_async_state.result)
-          async_state_done if form_response.success?
+
+          if form_response.success?
+            async_result_response = async_state_done(current_async_state.result)
+          end
```

Comment:
I'd like to make sure we still log the right stuff to analytics --- I was going to make a comment that we should `FormResponse#merge` the various form responses....but I don't know if any of them get returned here? How can we make sure this info is still logged?

Reply:
I added merge for this action and for the doc_auth_capture step. The step merge is a bit weird since it required doing some gymnastics to merge a doc auth response and form response 😬

Reply:
The merge *should* be fine --- I think we can get through most of it via duck typing/`#respond_to?` stuff

Reply:
The FSM checks if it's a `FormResponse` or `IdentityDocAuth::Response`, but we don't have a great way of combining those responses. Calling merge on one and passing in the other type doesn't work since they don't have the same structure. I think that behavior is correct though.